docs/userGuide/puml.md

```diff
@@ -0,0 +1,84 @@
+<variable name="title" id="title">PlantUML Diagrams</variable>
+<frontmatter>
+  title: "User Guide - {{ title }}"
+  layout: userGuide
+</frontmatter>
+
+
+# {{ title }}
+
+
+<span id="overview" class="lead">
+
+**MarkBind offers integration with PlantUML in your pages,** allowing you design and write UML diagrams (and more) using PlantUML syntax, which is then compiled and served as images.
```

Comment:
- Add a newline after the bold text?
- `(and more)` should naturally link to other examples
- You should put a warning at the top of this page indicating that authors need to have a JRE/JDK installed, along with the graphviz installer to use this feature.